curve25519Generate.go

```diff
@@ -1,4 +1,4 @@
-package curve25519
+package identity
```

Comment:
Renaming this package is arguably a good idea, but one pros for is to hide the following "internal" functions.

Reply:
IMHO, that would be a good idea. More info on internal packages: https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface

Reply:
Done, `internal` package created and `base64_json` and `crypto` were moved into it.